hs/test-examples/nl-eval-errors/Err_Sol_Reserved.txt

```diff
@@ -0,0 +1 @@
+reachc: error: ./Err_Sol_Reserved.rsh:5:24:application: `super` is not a valid field name because it is reserved in Solidity.
```

Comment:
Does the `_` prefix treatment not handle this use case? Do we intend for it to handle it in the future?

Reply:
It's not handled because the `Struct`s have to obey an interface with remote contracts